uaclient/status.py

```diff
@@ -392,8 +393,14 @@ def colorize_commands(commands: "List[List[str]]") -> str:
         if content:
             content += " && "
         content += " ".join(cmd)
+    # subtract 2 from print width to make room for space and backslash
+    wrapped_content = " \\\n".join(
```

Comment:
I don't think we need the double-escape squence here to we?
```suggestion
    wrapped_content = " \n".join(
```

Reply:
Ah so I added a backslash `\\` to the end of the line there, because thats how you would typically break up a command into multiple lines so that it can be copy-pasted.

E.g.
```
apt update && apt install package1 \
    package2 package3
```

This isn't intended to be copy-pasted/run by the user though, so we can get rid of it if you think its unnecessary.